modules/sb-configuring-ansible-service-broker-options.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+// * applications/service_brokers/configuring-ansible-service-broker.adoc
+
+[id='sb-ansible-service-broker-config-options-{context}']
+= {asb-name} configuration options
+
+You can set the following options for your {asb-name}.
+
+.{asb-name} configuration options
+[options="header",cols="1,2,1a"]
+|===
+|YAML key |Description |Default value
+
+|`brokerName`
+|The name used to identify the broker instance.
+|`ansible-service-broker`
+
+|`brokerNamespace`
+|The namespace where the broker resides.
+|`openshift-ansible-service-broker`
+
+|`brokerImage`
+|The fully qualified image used for the broker.
+|`docker.io/ansibleplaybookbundle/origin-ansible-service-broker:v3.11`
+
+|`brokerImagePullPolicy`
+|The pull policy used for the broker image itself.
+|`IfNotPresent`
+
+|`brokerNodeSelector`
+|The node selector string used for the broker's deployment.
+|`''`
+
+|`registries`
+|Expressed as a yaml list of broker registry configs, allowing the user to configure the image registries the broker will discover and source its APBs from.
+|Default registries array:
+----
+- type: dockerhub
```

Comment:
I'm not certain if there's anything that be done about this source block, but it doesn't wrap and renders quite poorly.

Reply:
I moved it below the table, let me know what you think!